packages/optimizer/spec/transformers/experimental/RewriteAmpUrls/adds_geoapi/expected_output.html

```diff
@@ -9,8 +9,8 @@
   <link rel="stylesheet" href="https://cdn.ampproject.org/v0.css">
   <link href="https://fonts.googleapis.com/css?foobar" rel="stylesheet" type="text/css">
   <link href="https://example.com/favicon.ico" rel="icon">
-  <link as="script" crossorigin="anonymous" href="https://cdn.ampproject.org/v0.mjs" rel="preload">
+  <link as="script" crossorigin="anonymous" href="https://cdn.ampproject.org/v0.mjs" rel="modulepreload">
```

Comment:
@erwinmombay are `as=script` and `crossorigin: 'anonymous'` required? `as=script` currently gives a validation error.

Reply:
It's not in prod which is why there is an error. See cl/363067094.

Reply:
The next release for JavaScript Validator should be tomorrow. @MichaelRybak please ping this thread when cl/363067094 goes live with the deployment.

Reply:
Neither is strictly necessary at the browser level. The validator will require both.

Reply:
Thanks for the clarification. Then I was just confused because `rel="modulepreload"` was already supported in the validator.